example/lib/main.dart

```diff
@@ -45,8 +46,11 @@ class _MyAppState extends State<MyApp> {
       print('startRecorder: $path');
 
       _recorderSubscription = flutterSound.onRecorderStateChanged.listen((e) {
-        DateTime date = new DateTime.fromMillisecondsSinceEpoch(e.currentPosition.toInt());
-        String txt = DateFormat('mm:ss:SS', 'en_US').format(date);
+        initializeDateFormatting();
```

Comment:
Does this should be called in every callback? What if just initializing this once?

Reply:
Hi Hyo Chan, Not required in every callback. Moved the initialization statement to init.